packages/docsearch-react/src/DocSearchModal.tsx

```diff
@@ -50,6 +50,7 @@ export function DocSearchModal({
   disableUserPersonalization = false,
   initialQuery: initialQueryFromProp = '',
   translations = {},
+  reportMissingResultsUrl,
```

Comment:
it's optional but there is are no default?
edit: the component is optional okay 🆗